packages/terra-core-docs/src/terra-dev-site/doc/progress-bar/About.1.doc.mdx

```diff
@@ -81,6 +82,7 @@ import ProgressBar from 'terra-progress-bar';
 
 ## Examples
 <ProgressBarDefault title="Default" />
+<ProgressBarValueTextProp title="Custom Text" />
```

Comment:
nit: Can we rename this
```suggestion
<ProgressBarCustomText title="Custom Text" />
```

Reply:
Changes Done. Thanks
90f0840b2961a617452496bbe974a240e66669dc